src/main/java/com/raisetech/mysql/sample/Controller/NameController.java

```diff
@@ -0,0 +1,22 @@
+package com.raisetech.mysql.sample.Controller;
+
+import com.raisetech.mysql.sample.service.NameService;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+@RestController
+public class NameController {
+    private NameService nameService;
+
+    public NameController(NameService nameService) {
+        this.nameService = nameService;
+    }
+
+    @GetMapping("/names")
+    public List<NameResponse> getName() {
```

Comment:
複数形のほうが実態に沿っていていいかと思います。
```suggestion
    public List<NameResponse> getNames() {
```

Reply:
geyNamesに変更しました。